plugins/CoreHome/javascripts/top_controls.js

```diff
@@ -64,4 +64,43 @@ function initTopControls() {
         }
 
     }
-}
\ No newline at end of file
+}
+
+//Tab controls for Top Controls calendar. 
+$( document ).ready(function() {
+    $('.periodSelector').keydown(function(e){
+
+        $('.periodSelector .ui-datepicker-month').attr('tabindex','4');
+        $('.periodSelector td a').attr('tabindex','4');
+        $('.periodSelector .ui-datepicker-year').attr('tabindex','4');
+        $('.periodSelector .form-radio').attr('tabindex','4');
+
+        var calendarOpen = $(this).hasClass('expanded');
+
+        if(e.which==13){
+            if(calendarOpen){
+                $(this).removeClass('expanded');
+            }else{
+                $(this).addClass('expanded');
+            }
+        }
+    })
+
+    $('.ui-datepicker-month, .ui-datepicker-year, .periodSelector td a').keydown(function(e){
```

Comment:
@Gabriel-Bowater  could you put this code into a function and re-use it below? this will help not duplicate the code